lib/getters/cartGetters.ts

```diff
@@ -10,8 +10,8 @@ export const getCartItemPrice = (
   item: CartItem
 ): { regular: Maybe<String> | undefined; special: Maybe<String> | undefined } => {
   return {
-    regular: `$${item.productDiscounts?.length ? item.subtotal : item.total}`,
-    ...{ special: `$${item.total}` },
+    regular: `$${item.product?.price?.price}`,
```

Comment:
Don't add the `$` here in the getters, just return the number do the formatting in the UI / component (use the $n formatter)

Reply:
Done